app/views/users/splash.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="w-full h-full" >
+
+  <div class="h-2/3 flex items-center justify-center">
+    <h2 class="font-bold text-primary text-2xl"> Smart Wallet </h2>
+  </div>
+
+  <div class="flex flex-col items-center justify-center gap-2">
+    <%= button_to "LOG IN", new_user_session_path, class: "bg-primary text-white px-7 py-2" %>
+    <%= button_to "SIGN UP", new_user_registration_path, class: "hover:bg-neutral-focus px-7 py-2" %>
+  </div>
```

Comment:
- [ ] As the requirements say, you design should look exactly the same as the one from the template. Kindly make sure you apply the same here on this welcome page, use the same color, fontfamily, font-size, layout,...

Your design | the template 
---|----
![image](https://github.com/stephenkati/Smart_Wallet/assets/65068771/b2be6566-9018-4956-8319-ba1cb22cc920) | ![image](https://github.com/stephenkati/Smart_Wallet/assets/65068771/6cf64cba-b1b3-4f72-b0be-4ce1cab2b54a)

Reply:
you are correct, my design does not look like the behance design. Thanks for the correction